app/parse.py

```diff
@@ -8,5 +10,32 @@ class Course:
     duration: str
 
 
+BASE_URL = "https://mate.academy/"
+
+
+def parse_single_course(soup: BeautifulSoup) -> Course:
```

Comment:
The `soup` parameter in `parse_single_course` is expected to be a BeautifulSoup object, but it is actually a single course element. Consider renaming the parameter to `course_element` to reflect its actual use and avoid confusion.